React Project/src/contexts/ToastContext.tsx

```diff
@@ -29,7 +29,7 @@ export function ToastContextProvider({ children }: { children: React.ReactNode }
         setColor(typeColor);
     }, [type]);
 
-    const toast = (message: string, type: MessageVariant) => {
+    const toast = (message: string, type: MessageVariant): undefined => {
```

Comment:
Why do we have to explicitly return `undefined` here?

Reply:
You're right. It makes more sense to use void here. Let me update it.